tools/errorprone/src/main/java/com/google/firebase/errorprone/ComponentsAppGetCheck.java

```diff
@@ -87,6 +87,19 @@
                   isSupertypeOf(
                       state -> ASTHelpers.getType(state.findEnclosing(ClassTree.class))))));
 
+  /**
```

Comment:
This needs review by someone on the Android Core team.

Reply:
Asked Vlad to review.